dips/dip-10.md

```diff
@@ -1,71 +1,64 @@
 
 ## Prerequisite:
-* VASPs get approval from association (via some offline process) on domain name
+* VASPs get approval from association (via some offline process) on domain name. 
+* VASPs have access to a storage solution mapping domain IDs to parent VASP address
 ```
 avasp, 0xf72589b71ff4f8d139674a3f7369c69b
 bvasp, 0xc5ab123458df0003415689adbb47326d
 ```
 
-## User Story: 
+## An Example P2P Transaction Flow: 
 * Bob wants to receive funds from Alice
-* Bob registers a Diem ID at VASP B: `bob@bvasp`
-* Alice's Diem ID is registered at VASP A: `alice@avasp`
-* Bob shares the Diem ID, `bob@bvasp` with Alice
+* Alice has an account with VASP A: `alice@avasp`
+* Bob has an account with VASP B: `bob@bvasp`
+* Bob shares the DiemID, `bob@bvasp` with Alice
 * Alice logs into VASP A, enters Bob’s identifier, an amount to pay, and submits payment
-* Alice’s VASP (VASP A) contacts Bob’s VASP’s (VASP B) off-chain API with the sender identifier `alice@avasp` and requests a reference_id, `rb`
+* VASP A contacts VASP B via an off-chain API with the sender identifier `alice@avasp` and requests a reference_id, `rb`
 * VASP A constructs a transaction with the specified amount and the reference_id `rb` and submits it to the Diem network
-* VASP B receives a transaction with metadata containing `rb`, deposits the amount in Bob's account, and attaches the relevant metadata (e.g., Alice’s Diem ID) to the internally stored transaction so Bob can confirm the transaction
+* VASP B receives a transaction with metadata containing `rb`, deposits the amount in Bob's account. (VASP B may use Alice’s DiemID for its own record keeping or seek receipt confirmation from Bob.)
 
 
-# Diem ID Format
-Diem defines globally unique identifiers to facilitate transacting on the Diem Payment Network. As Diem stores no PII on-chain, these identifiers are exchanged during the off-chain process between two VASPs to identify a distinct source and destinations within their respective VASPs in order to produce a reference_id that can be stored on-chain. The format for the identifiers follows:
-`[user_identifier]@[vasp_domain_identifier]`
+# DiemID Format
+Diem defines globally unique identifiers to facilitate transacting on the Diem Payment Network. As Diem stores no personally identifiable information on-chain, these identifiers are exchanged during the off-chain process between two VASPs to identify a distinct source and destinations within their respective VASPs in order to produce a reference_id that can be stored on-chain. The format of identifiers is `[user_identifier]@[vasp_domain_identifier]`
 
 Example: `alice@avasp`
 
 * `user_identifier` is a reusable identifier that represents either the source or destination end-user of a payment transaction. It is unique to per user at VASP level. Specification:
   * Case insensitive
-  * Valid regular expression: `[a-zA-Z0-9]+`
+  * Valid regular expression: `^[a-zA-Z0-9][a-zA-Z0-9.]*$`
```

Comment:
it might be worthwhile having a section at the bottom describing how we settled on this, how compatible is it with other naming schemes?